rclcpp/include/rclcpp/node.hpp

```diff
@@ -1448,6 +1449,51 @@ class Node : public std::enable_shared_from_this<Node>
   rclcpp::node_interfaces::NodeTimeSourceInterface::SharedPtr
   get_node_time_source_interface();
 
+  /// Return a NodeInterfaces bound with the Node's internal node interfaces.
+  /**
+   * Specify which interfaces you want to bind using the template parameters by specifying
+   * interface support classes to use. Any unmentioned interfaces will be unavailable to bind.
+   *
+   * This method will return a NodeInterfaces with no bound interfaces. You must set them using
```

Comment:
I think this is a bit strange, I would expect it to be the same as `NodeInterfaces<...>(node)`, I wouldn't expect it to work like the default constructor.

Reply:
Actually, the implementation is what I expect.
I think either the documentation is outdated or not super clear

Reply:
https://github.com/ros2/rclcpp/pull/2041/commits/2d65b59c855d73b66ecbdf5638b59d408fc7f7b9